src/controllers/post.controller.ts

```diff
@@ -28,7 +29,7 @@ export const createPost = [
     const createPostDto: CreatePostDto = req.body;
     await postService.create(createPostDto, currentUserId);
 
-    res.redirect(`/posts/fyp/1`);
+    res.redirect(`/posts/fyp?page=1`);
```

Comment:
ko cần truyền params page đâu e
lúc nào có phân trang thì mới cần xử lý thêm
```suggestion
    res.redirect(`/posts/fyp`);
```